README.md

```diff
@@ -27,21 +26,20 @@ For Cooks, the platform provides a simple way to create profiles and showcase th
 
 1.	Connects customers with local cooks offering homemade food.
 2.	Browse and explore a diverse range of homemade dishes.
-3.	User-friendly interface for placing and managing orders.
-4.	Profiles for cooks to showcase their culinary offerings.
-5.	Order tracking and communication between customers and cooks.
+3.	Profiles for cooks to showcase their culinary offerings.
+4.	Order tracking and communication between customers and cooks.
 
 ## Installation
 
-To install ChatGPT Devs Era, you can use yarn:
+To install Cookiz, you can :
+Setup-cookiz app and we can use this steps:
```

Comment:
You can use this instead, I think it's better `To set up cookiz app do the following steps:`